prometheus/src/main/scala/filodb/prometheus/ast/Vectors.scala

```diff
@@ -191,7 +193,8 @@ trait Vectors extends Scalars with TimeUnits with Base {
         throw new UnsupportedOperationException("Range expression is not allowed in query_range")
       }
       // multiply by 1000 to convert unix timestamp in seconds to millis
-      RawSeries(timeParamToSelector(timeParams, window.millis), columnFilters, column.toSeq)
+      RawSeries(timeParamToSelector(timeParams, window.millis, offset.map(_.millis).getOrElse(0)), columnFilters,
```

Comment:
Can offset be requested for raw data query ?

Reply:
Yes offset can be used for raw data query also